specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.5-preview.1/backuprestore.json

```diff
@@ -82,6 +82,61 @@
         }
       }
     },
+    "/managedidentitymodebackup": {
+      "post": {
+        "tags": [
+          "ManagedIdentityModeFullBackup"
+        ],
+        "operationId": "ManagedIdentityModeFullBackup",
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        },
+        "description": "Creates a full backup using a user-provided Azure blob storage container uri.",
+        "parameters": [
+          {
+            "in": "body",
+            "name": "azureStorageBlobContainerUri",
+            "schema": {
+              "type": "string"
+            },
+            "description": "Azure blob storage container uri"
+          },
+          {
+            "$ref": "common.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Started full backup",
+            "headers": {
+              "Retry-After": {
+                "description": "The recommended number of seconds to wait before calling the URI specified in Azure-AsyncOperation.",
+                "type": "integer"
+              },
+              "Azure-AsyncOperation": {
```

Comment:
Data plane APIs should use Operation-Location: https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#obtaining-status-and-results-of-long-running-operations